airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationConfig.java

```diff
@@ -127,6 +141,10 @@ public S3FormatConfig getFormatConfig() {
   }
 
   public AmazonS3 getS3Client() {
+    if(this.s3Client !=null) {
```

Comment:
formatting: space between `=` and `null` 

this should happen automatically if you run `./gradlew format` . Also it should happen automatically in intellij if you follow these instructions https://docs.airbyte.com/contributing-to-airbyte/code-style

Reply:
done